zookeeper-api/src/main/java/org/apache/helix/zookeeper/routing/RoutingDataManager.java

```diff
@@ -183,6 +199,25 @@ public long getLastResetTimestamp() {
     return _lastResetTimestamp;
   }
 
+  /**
+   * Resolves the routing data update interval value from System Properties.
+   */
+  public void parseRoutingDataUpdateInterval() {
```

Comment:
It's public for testing purpose; and technically, this function is a setter (there may be necessity to refresh the interval). Open for suggestions.

Reply:
Got it, if there is a need for runtime adjustment (in real use case), it might be better as a simple setter. We can separate out the config reading part. 
No strong preference though.

Reply:
The parsing logic is universal, though. I will keep it the current way; the other approaches don't seem clean.